mne/time_frequency/tfr.py

```diff
@@ -2064,12 +2176,30 @@ def __init__(self, info, data, times, freqs, comment=None, method=None,
         if events is None:
             n_epochs = len(data)
             events = _gen_events(n_epochs)
+        if selection is None:
+            n_epochs = len(data)
+            selection = np.arange(n_epochs)
+        if drop_log is None:
+            drop_log = tuple(
+                () if k in selection else ('IGNORED',)
+                for k in range(max(len(events),
+                               max(selection) + 1)))
```

Comment:
this is a little unwieldy.  can you pull `max(len(events), max(selection) + 1)` out as a variable with an intelligible name?

Reply:
```suggestion
            n_epochs_prerejection = max(len(events), max(selection) + 1)
            drop_log = tuple(
                () if k in selection else ('IGNORED',)
                for k in range(n_epochs_prerejection))
```
better? 🙂